modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -344,17 +352,34 @@ const firstColumnIsLocked = (_warehouse: Warehouse, details: Structs.DetailExt[]
 const lastColumnIsLocked = (warehouse: Warehouse, details: Structs.DetailExt[]) =>
   Arr.exists(details, (detail) => detail.column + detail.colspan >= warehouse.grid.columns && detail.isLocked);
 
-const insertColumnExtractor = (before: boolean) => (warehouse: Warehouse, target: RunOperation.TargetElement): Optional<Structs.DetailExt> =>
+const getPixelDelta = (details: Structs.DetailExt[]) => {
```

Comment:
Maybe call this `getDetailWidths` as that's closer to what this is doing. The delta part is really specific for columns.